tests/conftest.py

```diff
@@ -235,6 +235,12 @@ def application_docker_image_configs() -> Mapping[str, Dict[str, Any]]:
         },
         "java": {
             "": {},
+            "hotspot-jdk-8": {},  # add for clarity when testing with multiple JDKs
+            "hotspot-jdk-11": dict(
+                buildargs={
+                    "JAVA_BASE_IMAGE": "openjdk@sha256:e81b7f317654b0f26d3993e014b04bcb29250339b11b9de41e130feecd4cd43c"
```

Comment:
```suggestion
                    "JAVA_BASE_IMAGE": "openjdk/11-jdk"
```
or similar does not work?

Reply:
The hash is when we want a precise build (i.e matching buildid, etc). IDK if needed in this case